### ISSUE TITLE
Add logger

### ISSUE THREAD

**Issue by joelnet** [joelnet/the-terminal-daemon#5]
create `src/logger.js` and export a Winston logger

Winston: https://www.npmjs.com/package/winston

The logs should output to the console and to `.logs/`

log level should be read from config.

Also use `winston-daily-rotate-file` to rotate the log daily and limit the amount of logs collected to 7 days.

At a minimum, log the startup of the app, a users login even and the unhandled shutdown of the app in `src/index.js`.

```javascript
process
  .on('uncaughtException', err => {
    // TODO: add logging logic here
    process.exit(1);
  });

```

**Comment by JeffMartian** [comment]
Hi @joelnet, I'd like to work on this.

**Comment by joelnet** [comment]
👍

**Comment by joelnet** [comment]
code has been merged in #9

**Comment by joelnet** [comment]
@all-contributors please add @DaveSalazar  for code
